### PR TITLE
Adds upcoming features to top of README.

### LINKED ISSUE #69: readme should make it clear that this is for non-war use-cases at the very top

### PR DESCRIPTION
**Pull request by coollog**
Fixes #69 

Named this `Upcoming Features`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ Jib is a Maven plugin for building container images for your Java applications.
 
 * **Native** - Reduce your CLI dependencies. Build your Docker image from within Maven <!--or Gradle--> and push to any registry of your choice. No more writing Dockerfiles and calling docker build/push.
 
+## Upcoming Features
```
Comment (loosebazooka) [inline README.md]: I would put this in later, maybe even at the end?

**Comment by loosebazooka** [comment]
@parkjam4 hey jimmy, what do you think of this format? We want to tell users that only `JAR`s  are supported now and not `WAR`s so we're including this section in here to tell everyone what we don't support yet (but in a nice positive coming-soon way :smile:).

File: README.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ Jib is a Maven plugin for building container images for your Java applications.
...
+
+These features will be added in later releases.
```
Comment (parkjam4) [inline README.md]: I know what you're trying to do here, but the wording feels a little strange to me.
What if you were more explicit that these features are not supported?
Like changing this line to:
These features are currently not supported but will be coming in later releases.

Comment (coollog) [reply README.md]: Okay added the `currently not supported` phrase!

**Pull request merged**